scripts/ansible/roles/linux/intel/vars/redhat/ootpa.yml

```diff
@@ -20,3 +20,4 @@ intel_dcap_packages:
   - "libsgx-dcap-ql"
   - "libsgx-dcap-ql-devel"
   - "libsgx-urts"
+  - "libsgx-quote-ex"
```

Comment:
Do we need AESM plugins too?

Reply:
To use libsgx-quote-ex with ECDSA quoting (out-of-proc) to the AESM you need many packages.  I have listed the packages with their dependencies below them.  The dependencies are hard dependencies (the package installer will either pull these dependencies from the repo and install them or require them to be installed in order to install the package)  unless noted as an optional dependency (though I have listed the optional dependencies that you need).  The process would be to install dependencies first:
- libsgx-aesm-quote-ex-plugin (optional dependency, but you need it) - depends on:
- libsgx-aesm-service
- libsgx-aesm-ecdsa-plugin (optional dependency, but you need it) - depends on:
 		- sgx-aesm-service
- libsgx-qe3-logic – depends on:
- libsgx-ae-qe3
		- libsgx-aesm-pce-plugin – depends on: 
			- sgx-aesm-service
			- libsgx-pce-logic – depends on:
				- libsgx-ae-pce

I have left off the dependencies on libsgx-enclave-common and libsgx-urts which you should install first.

If you use the libsgx-dcap-ql, then this depends on libsgx-quote-ex and also depends on libsgx-dcap-quote-verify which depends on libsgx-ae-qve.

Reply:
`libsgx-aesm-service` and three `libsgx-aesm-*-plugin`s have been added to both installation scripts. I left out the `libsgx-pce-*` and `libsgx-qe3-*` ones because they are already listed on the main installation files.

Reply:
From Mark's list, it looks like "sgx-aesm-service" should go all the plugins.

Reply:
sgx-aesm-service does not install plugins, but libsgx-quote-ex did install all three libsgx-aesm-* plugins for me.